tools/docker/ts-integration.Dockerfile

```diff
@@ -20,27 +20,18 @@ ENV PATH=/chainlink/tools/bin:./node_modules/.bin:$PATH
 ARG SRCROOT=/usr/local/src/chainlink
 WORKDIR ${SRCROOT}
 
-COPY yarn.lock package.json .yarnrc ./
-COPY .yarn .yarn
-COPY belt/package.json ./belt/
-COPY belt/bin ./belt/bin
-COPY evm-test-helpers/package.json evm-test-helpers/
-COPY evm-contracts/package.json ./evm-contracts/
-COPY tools/ci-ts/package.json tools/ci-ts/
-
-# copy our CI test
-COPY tools/docker tools/docker/
-
 # copy over all our dependencies
-COPY tsconfig.cjs.json tsconfig.es6.json ./
+COPY yarn.lock package.json .yarnrc tsconfig.cjs.json tsconfig.es6.json ./
+COPY .yarn .yarn
```

Comment:
Can this be added to the line above?

Reply:
this is a folder, so I guess I tried to keep all the folders separate from the config files above. That seem reasonable?